From aa044c1068d281bce2d0465874c256eb1da4c60a Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@altera.com>
Date: Tue, 6 Aug 2013 17:28:38 -0500
Subject: [PATCH 06/13] drivers/clk/socfpga: Fix incorrect clock value for
 SD/MMC

commit 00c11a46356143ccc511f33a38efe516f824c576
git://git.rocketboards.org/linux-socfpga.git socfpga-3.10-ltsi branch

The clock driver was not able to get the correct clock value for the
sdmmc_clk that feeds the CIU(Card Interface Unit). The name of the clock
clock that feeds the CIU should be "sdmmc_clk", NOT "mmc_clk". Hence,
the driver was using an an incorrect clock value. This lead to a write
performance degrade.

Signed-off-by: Dinh Nguyen <dinguyen@altera.com>
Signed-off-by: Yang Wei <Wei.Yang@windriver.com>
---
 drivers/clk/socfpga/clk.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/clk/socfpga/clk.c b/drivers/clk/socfpga/clk.c
index a40bfdd..c770441 100644
--- a/drivers/clk/socfpga/clk.c
+++ b/drivers/clk/socfpga/clk.c
@@ -50,7 +50,7 @@
 #define SOCFPGA_L4_SP_CLK			"l4_sp_clk"
 #define SOCFPGA_NAND_CLK		"nand_clk"
 #define SOCFPGA_NAND_X_CLK		"nand_x_clk"
-#define SOCFPGA_MMC_CLK			"mmc_clk"
+#define SOCFPGA_MMC_CLK			"sdmmc_clk"
 #define SOCFPGA_DB_CLK			"gpio_db_clk"
 
 #define div_mask(width)	((1 << (width)) - 1)
-- 
1.7.5.4

